packages/pds/src/app-view/services/indexing/plugins/post.ts

```diff
@@ -143,38 +143,48 @@ const findDuplicate = async (): Promise<AtUri | null> => {
 
 const eventsForInsert = (obj: IndexedPost) => {
   const notifs: Message[] = []
+  const notified = new Set([obj.post.creator])
+  const maybeNotify = (notif: messages.NotificationInfo) => {
```

Comment:
nice 👍 

so one notif per post & order of precendence goes:
- mention
- quote
- reply